packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart

```diff
@@ -13,19 +13,45 @@ String generateWasmBootstrapFile() {
   let moduleInstance;
   try {
     const dartModulePromise = WebAssembly.compileStreaming(fetch("main.dart.wasm"));
+    ${generateImports(isSkwasm)}
     dart2wasm_runtime = await import('./main.dart.mjs');
-    moduleInstance = await dart2wasm_runtime.instantiate(dartModulePromise, {});
+    moduleInstance = await dart2wasm_runtime.instantiate(dartModulePromise, imports);
   } catch (exception) {
-    console.error(`Failed to fetch and instantiate wasm module: ${exception}`);
+    console.error(`Failed to fetch and instantiate wasm module: \${exception}`);
```

Comment:
Wow, how did this compile before? I don't see a variable named `exception` in scope 🤔

Reply:
In this PR, we changed this from a raw string literal (i.e. `r'''`) to a multi-line string literal (i.e. `'''`) so we didn't need to escape before. We are doing string interpolation depending on whether you are going to actually import skwasm or not, so we needed to switch away from the raw string literal and escape any `$` in the document.